swarm/storage/hasherstore.go

```diff
@@ -83,6 +92,11 @@ func (h *hasherStore) Put(ctx context.Context, chunkData ChunkData) (Reference,
 	chunk := h.createChunk(c)
 	h.storeChunk(ctx, chunk)
 
+	// Start the wait function which will detect completion of put
+	h.doWait.Do(func() {
+		go h.startWait(ctx)
```

Comment:
this is not needed, we have tags now

Reply:
I am not aware of tags. will look in to it and remove it.

Reply:
Done